packages/facebook-config-ui/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: "Blaze configuration templates for Facebook OAuth.",
```

Comment:
I guess it's not necessary, but do these need `name`?  Downsides?  Upsides?  :shrug: